client/src/components/Json.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    <div class="p-1 font-mono">
+      <div v-if="typeof value === 'object'">
+        <button class="flex items-center" @click="open = !open">
+          "{{ name }}"<span>: {</span>
+        </button>
+        <ul
+          v-if="open"
+          class="pl-1"
+          :class="{ 'ml-2.5 border-l border-dotted': open }"
+        >
+          <li v-for="(item, key) in value">
```

Comment:
Mein Linter hätte gerne :key in dieser Zeile.
Würde das aber auch so durchgehen lassen. Ich weiß nicht genau was technisch dahinter steckt - ob Vue das irgendwann braucht um ein Element eindeutig zu identifizieren. 
> Elements in iteration expect to have 'v-bind:key' directives.eslint-plugin-vue

Reply:
👍